Release 2.2.0-beta.1 (#1520)
finos/FDC3: README.md

For installation and usage instructions, see: <https://fdc3.finos.org/docs/supported-platforms#usage>

> ### Releasing FDC3 (for maintainers)

Reviewer: This is only covering the NPM module release, rather than also Standard website
```suggestion
### Releasing FDC3 NPM modules (for maintainers)
```

This is a 4-step process:

1.  **Create a release branch**

  Do this locally.  Ensure the branch is named `release/v2.0` (or whatever the next version is).
  
  ```bash
  git checkout -b release/v2.0
  ```
  
2.  **Update the version numbers in the package.json files**

  It's important that all of the versions of the submodules stay on the same version, and that the references between them are consistent to that version.  To change the version number (say before or after a release) run the following:

  ```bash
  // first, update version number in package.json
  npm login
  npm version <new version from package.json> --workspaces // changes the version number in all submodule package.json files
  npm run syncpack                          // sycnhronizes version numbers
  npm up                                    // fixes node_module references
  npm run build                             // builds all the modules against the new version
  ```

3.  **Push the branch to publish the packages to npm**

  ```bash
  git push origin release/v2.0
  ```
  
  This should trigger a GitHub action that will publish the packages to npm.

4.  **Create a PR for merging the release branch.**

>   Once the packages are published, create a PR to merge the release branch into the main branch.  You will need FDC3 maintainers to review and approve the PR.

Reviewer: ```suggestion
  Once the packages are published, create a PR to merge the release branch into the main branch.  You will need other FDC3 maintainers to review and approve the PR.
```

> 5.  **Set The NPM Version If Releasing A Beta**

Reviewer: ```suggestion
5.  **Reset the `latest` NPM Version Tag If Releasing A Prerelease**
```

  If you're releasing beta/alpha code, be sure to replace the latest version in NPM like so:

  ```

>   npm dist-tag add @finos/fdc3@2.1.1 latest

Reviewer: Project maintainers will not be able to run `npm dist-tag` due to not having permissions on the NPM project, so it might be worth inserting a note here about contact help@finos.org or yourself to sort that out...

Its that of we detect that there is something after the last digit of the version number in the workflow itself and have it set an alternative tag when it runs `npm publish`, e.g. `npm publish --tag prerelease`. We can probably figure out a regex to extract 'beta' from 2.2.0-beta.1 automatically...
